runtime/frequency/src/lib.rs

```diff
@@ -521,6 +521,7 @@ impl pallet_collator_selection::Config for Runtime {
 parameter_types! {
 	pub const MaxMessagesPerBlock: u32 = 7000;
 	pub const MaxMessagePayloadSizeBytes: u32 = 1024 * 50; // 50K
+	pub const MaxOffchainPayloadSizeBytes: u32 = 1024 * 128;
```

Comment:
I'm not totally sure if this is correct. DSNP spec has "Batch files MUST have NO MORE THAN 128*1024 rows." it doesn't say anything about row size. So We are talking about max payload size I'm not sure this is correct

Reply:
Good point. Maybe we dont need to check a max offchain payload size right now then?